src/util/languages/languages.js

```diff
@@ -0,0 +1,99 @@
+import CreateProcessingDoc from "./Processing";
+import CreatePythonDoc from "./Python";
+import CreateReactDoc from "./React";
+import { faCogs } from "@fortawesome/free-solid-svg-icons";
+import { faPython, faJs, faHtml5, faReact } from "@fortawesome/free-brands-svg-icons";
+
+export const SUPPORTED_LANGUAGES = [
+  {
+    value: "python",
+    display: "Python",
+    icon: faPython,
+    codemirror: "python",
+    extension: "py",
+    render: (code, showConsole) => CreatePythonDoc(btoa(code), showConsole),
+  },
+  {
+    value: "javascript",
+    display: "JavaScript",
+    icon: faJs,
+    codemirror: "javascript",
+    extension: "js",
+  },
```

Comment:
Only comment here is that JavaScript is actually not officially supported by the editor. We could easily add this, but it'd have to be in another series of PRs since the backend won't accept this.